### ISSUE TITLE
Build failed when the DesignTools assemblies are loaded by Visual Studio.

### ISSUE THREAD

**Issue by benZhai01** [microsoft/xaml-designer-extensibility#13]
I'm sorry that I can't find a correct place to report the issues of xaml design time. if any suggestions, please let me know☺.

### Environment:
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.4.0 Preview 5.0

### Issuse:
After opening the `MainWindow.xaml` in `DemoApplication` project(building the entire solution successfully before, so I think the DesignTools is loaded by VS), then I modify some code in `CustomControlLibrary.WpfCore.DesignTools` project and want to re-build it, but failed.

![image](https://user-images.githubusercontent.com/47809203/69318098-69916800-0c77-11ea-8dd0-e11b0e522fcb.png)

Is there any way to improve this? because it's really a nightmare during developing the XAML design time.

**Comment by hemantbajpai4** [comment]
VS loads assembly and it gets locked on disk so we get build errors. Workaround is to close VS instance and open it again then build should work.

**Comment by benZhai01** [comment]
@hemantbajpai4 
> VS loads assembly and it gets locked on disk so we get build errors. Workaround is to close VS instance and open it again then build should work.

Yeah, that's a workaround, but it's weird and really inconvenient to me, can we have another better option? it's impossible?
Seems the old architecture won't have the problem, the new one should not at least be worse than the old one, right?

**Comment by benZhai01** [comment]
Any updates?

**Comment by benZhai01** [comment]
I spent almost half of the time to restart VS when developing desgin time assemblies. Can anyone address it?

**Comment by hemantbajpai4** [comment]
We will investigate this and update you, currently I am reopening this issue. Thanks

**Comment by dongyaosheng** [comment]
Can I release the occupation of the corresponding designtool.dll when the view is closed?
